<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoxReady</class>
 <widget class="QWidget" name="BoxReady">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Home_BoxReady_btStart">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Home_BoxReady_btPen">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Home_BoxReady_btCancelTest">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_BoxReady_lbBK1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_BoxReady_lbBK2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_BoxReady_lbTestName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_BoxReady_lbTestSerial">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Home_BoxReady_cbTestType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Home_BoxReady_lbSampleCode">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_BoxReady_lbSampleInfo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_BoxReady_lbSampleRemark">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
